nextflow_schema.json

```diff
@@ -99,6 +99,78 @@
+                    "fa_icon": "fab fa-accusoft",
+                    "description": "Specifies the kingdom that the input represents.",
+                    "help_text": "Specifies the kingdom that the input represents. Attention: PROKKA cannot annotate Eukaryotes.\n\n> Modifies tool parameter(s):\n> - PROKKA: `--kingdom`",
+                    "enum": ["Archaea", "Bacteria", "Mitochondria", "Viruses"]
+                },
+                "annotation_prokka_gcode": {
+                    "type": "integer",
+                    "default": 0,
+                    "minimum": 0,
+                    "maximum": 25,
+                    "description": "Specifies the translation table used to annotate the sequences.",
+                    "help_text": "Specifies the translation table used to annotate the sequences. All possible genetic codes (1-25) used for gene annotation can be found [here](https://en.wikipedia.org/wiki/List_of_genetic_codes). This flag is required if the flag `--kingdom` is assigned. \n\n> Modifies tool parameter(s):\n> - PROKKA: `--gcode`",
+                    "fa_icon": "fas fa-border-none"
+                },
+                "annotation_prokka_mincontiglen": {
+                    "type": "integer",
+                    "default": 1,
+                    "description": "Limit the contigs lengths.",
+                    "help_text": "Limit the contig lengths to only carry out annotations on contigs >= the assigned length. This should not be >200 as NCBI can only process contigs >= 200.\n\n> Modifies tool parameter(s):\n> - PROKKA: `--mincontiglen`",
+                    "fa_icon": "fas fa-ruler-horizontal"
+                },
+                "annotation_prokka_evalue": {
+                    "type": "number",
+                    "default": 1e-6,
+                    "description": "Minimum e-value cut-off.",
+                    "help_text": "Assigns the minimum e-value used for filtering the alignment hits.\n\n> Modifies tool parameter(s):\n> - PROKKA: `--evalue`",
+                    "fa_icon": "fas fa-sort-amount-down"
+                },
+                "annotation_prokka_coverage": {
+                    "type": "integer",
+                    "default": 80,
+                    "description": "Set the assigned minimum coverage.",
+                    "help_text": "Specifies the minimum coverage of the annotated genome. This must be set between 0-100. \n\n> Modifies tool parameter(s):\n> - PROKKA: `--coverage`",
+                    "fa_icon": "fas fa-align-right"
+                },
+                "annotation_prokka_cdsrnaolap": {
+                    "type": "boolean",
+                    "description": "Allow transfer RNA (trRNA) to overlap coding sequences (CDS). Transfer RNAs are short streatches of nucleotide sequences that link mRNA and teh amino acid sequence of proteins. Their presence helps in the annotation of the sequences, because each trRNA can only be attached to one type of amino acid.",
```

Comment:
```suggestion
                    "description": "Allow transfer RNA (trRNA) to overlap coding sequences (CDS). Transfer RNAs are short stretches of nucleotide sequences that link mRNA and the amino acid sequence of proteins. Their presence helps in the annotation of the sequences, because each trRNA can only be attached to one type of amino acid.",
```